File: script-engine-benchmark-micro/src/main/resources/com/netcracker/mediation/scripting/benchmark/runner/simple/eval/tpk.py
from java.lang import Math

def operation(number, index):
    return number + 5*index*index

def getRandomInt(min, max):
    return Math.floor(Math.random() * (max - min + 1)) + min

numbers = []
for i in xrange(11):
    numbers.append(getRandomInt(0, 500))
    print "> " + str(numbers[i]) + "\n"

for i in xrange(10, -1, -1):
    print "for x = " + str(i) + "y "
    result = operation(numbers[i], i)
    if result > 400:
        print "IS TOO LARGE"
    else:
        print "= " + str(result) + "\n"

File: script-engine-benchmark-micro/src/main/resources/com/netcracker/mediation/scripting/benchmark/jvm/runner/simple/eval/tpk.js
function operation(number, index) {
    return number + 5*index*index;
}

function getRandomInt(min, max) {
    return Math.floor(Math.random() * (max - min + 1)) + min;
}

var numbers = [];
for (var i = 0; i < 11; i++) {
    numbers[i] = getRandomInt(0, 500);
    print("> " + numbers[i] + "\n");
}
for (var i = 10; i >= 0; i--) {
    print("for x = " + i + "y ");
    var result = operation(numbers[i], i);
    if (result > 400) {
        print("IS TOO LARGE");
    } else {
        print("= " + result + "\n");
    }
}